src/Services/Air/Air.js

```diff
@@ -15,6 +15,20 @@ module.exports = (settings) => {
       return service.searchLowFares(options);
     },
 
+    fareRules(options) {
```

Comment:
Do fare rules could be different for different passengers types?
I think this should be an input param.

Please check with Travelport helpdesk before continue.

Reply:
That is correct, but the only planned use case for this method was to look up fare rules from segments taken from search results. Because of current search output format limitations (`fareBasisCode` being part of segments, not separate `bookingComponents`) it is impossible to have a search result that uses different fare bases for different PTCs. This is a serious limitation and it should be addressed in another Issue/PR and could lead to change in major version number, because of change in interface.
If a user wants to look up fare rules for an existing booking, I suggest using the native uAPI `FareRulesReq` which is not implemented now (there's an unused template commited in XML templates).

Reply:
And, you're right, we should change the request format to be analogous to search request.

Reply:
@articice , please create the ticket to reflect the issue described above.

Also, please, change usage of `_` to native JS functions before continue.

Reply:
Already did it.
Give me 1 hour to rewrite the request params.